src/Apps/Order/Components/TransactionDetailsSummaryItem.tsx

```diff
@@ -88,19 +88,26 @@ export const TransactionDetailsSummaryItem: FC<TransactionDetailsSummaryItemProp
     return useLastSubmittedOffer ? order.lastOffer : order.myLastOffer
   }
 
+  const getShippingLabel = shippingQuoteType => {
+    const artsyShippingLabel =
+      shippingQuoteDisplayNames[shippingQuoteType] + " delivery"
+    const suffix = offerShippingCostSubjectToChange() ? "*" : ""
+    return artsyShippingLabel + suffix
+  }
+
   const shippingDisplayLabel = shippingNotCalculated => {
     if (shippingNotCalculated()) {
       return "Shipping**"
```

Comment:
wondering if we could address this ** here too? it would be something like 
```suggestion
      return "Shipping*"
```
here and in the message + a check for `shippingNotCalculate() || offerShippingCostSubjectToChange()` in the tax logic to show the dagger.
WDYT?

Reply:
On it!

Reply:
Adjusted -- Keeps the pattern going if there is ever 2 footnotes to use `*` and then `†`

Ex with shipping not calculated flow:
<img width="1920" alt="Screen Shot 2023-05-26 at 9 35 30 AM" src="https://github.com/artsy/force/assets/12748344/33934e2b-97df-4555-9176-9507c4477c09">